openapiart/tests/test_set.py

```diff
@@ -0,0 +1,117 @@
+import sys
+import os
+import pytest
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", "art"))
+import sanity
+
```

Comment:
I'll suggest couple of things here:
- we should serialize at the end to make sure we don't see any exceptions
- we should add some negative cases, i.e. try setting some invalid value or type, it should throw exception (either during set or serialize)

Reply:
submitted the changes is test